DASCharter-2020.html

```diff
@@ -994,8 +994,8 @@ <h3>
             Group</a>
           </dt>
           <dd>
-            This group defines relevant specifications including <cite><a href=
-            "https://www.w3.org/TR/FileAPI/">File API</a></cite> and <a href=
+            This group defines relevant specifications including <a href=
```

Comment:
Add "the" before "File API"?

Reply:
Fix https://github.com/w3c/dap-charter/pull/85/commits/9f6cc2cb9bd81d4272b62aed2165479618af14d1